bokeh/server/tornado.py

```diff
@@ -252,8 +248,28 @@ def __init__(self, applications, prefix, hosts,
 
         super(BokehTornado, self).__init__(all_patterns)
 
+    def initialize(self,
+                 io_loop=None,
+                 keep_alive_milliseconds=37000,
+                 # how often to check for unused sessions
+                 check_unused_sessions_milliseconds=17000,
+                 # how long unused sessions last
+                 unused_session_lifetime_milliseconds=15000,
+                 # how often to log stats
+                 stats_log_frequency_milliseconds=15000,
+                 develop=False,
+                 **kw):
+
+        if io_loop is None:
+            io_loop = IOLoop.current()
+        self._loop = io_loop
+
+        for app_context in self._applications.values():
+            app_context._loop = self._loop
+
         self._clients = set()
         self._executor = ProcessPoolExecutor(max_workers=4)
+        print(self._loop)
```

Comment:
ha, I had removed and introduced it back in when I rebased. I'll take it back out